README.md

```diff
@@ -16,20 +16,25 @@ We welcome your contribution! Please see our
 You want to run this bad boy locally?  Let's do it.  
 
 ###Install Tools
--Install nodejs from https://nodejs.org/en/
+* Install nodejs from https://nodejs.org/en/
 
--Install npm from https://www.npmjs.com/
+* Install npm from https://www.npmjs.com/
+ * It is suggested that you use [yarn](https://yarnpkg.com/)
+  as an alternative to `npm` for package management. With
+  `npm`, you can instll this as in `npm install -g yarn`.
```

Comment:
Typo here 'instll'. Might want to reword. You can install yarn with the following command: `npm install -g yarn`.